code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -15,9 +15,9 @@
 	var/spin_delay = 10
 	var/recent_spin = 0
 
-/obj/item/gun/ballistic/revolver/chamber_round(spin = 1)
+/obj/item/gun/ballistic/revolver/chamber_round(spin = TRUE)
```

Comment:
```suggestion
/obj/item/gun/ballistic/revolver/chamber_round(keep_bullet, spin_chamber = TRUE)
```

Reply:
chamber_round() is fully overriden with no super call, keep_bullet would be nonfunctional